trikot-viewmodels-declarative/sample/common/TRIKOT_FRAMEWORK_NAME.podspec

```
@@ -4,8 +4,8 @@ Pod::Spec.new do |spec|
     spec.homepage                 = 'www.mirego.com'
     spec.source                   = { :git => "Not Published", :tag => "Cocoapods/#{spec.name}/#{spec.version}" }
     spec.authors                  = ''
-    spec.license                  = 'No license'
-    spec.summary                  = 'Awesome library'
+    spec.license                  = { :type => 'BSD' }
```

We should use `BSD-3` here.

```suggestion
    spec.license                  = { :type => 'BSD-3' }
```